spec/models/requests/requestable_spec.rb

```diff
@@ -835,8 +797,8 @@
     end
 
     describe "#available?" do
-      it "is not available" do
-        expect(requestable).not_to be_available
+      it "is available" do
```

Comment:
I'm unsure why an on order item would be considered available

Reply:
The items are requestable by the patron, right now we show on order titles as "grey" in user interface.

Reply:
@kevinreiss The concept of available is that the item is available right now.  Aka on the shelf.  I do not think these items should show as available Green right now.  There is a concept of In Process, which is what these items should be.